test/e2e/network/service.go

```diff
@@ -3961,6 +3961,7 @@ var _ = SIGDescribe("SCTP [Feature:SCTP] [LinuxOnly]", func() {
 	})
 
 	ginkgo.It("should create a Pod with SCTP HostPort", func() {
+		e2eskipper.RunIfContainerRuntimeIs("docker")
```

Comment:
I’m just wondering why we can’t handle that via the ginkgo test scope.

Reply:
https://github.com/kubernetes/kubernetes/pull/92079#issuecomment-643489840
we should avoid these kind of skips in the k/k framework, each CRI can implement their own tests in their infra/repo IMHO

Reply:
Since we hope this is going to be fixed in the runtimes soon, maybe we should wait and only do the skip if the test fails? Then as soon as the CI environment gets an updated containerd, the test will start passing (and then eventually we can remove the skip).

Reply:
(by "wait", I mean, instead of skipping here, do it before the `framework.Failf` in the `if !found {` block below)

Reply:
@danwinship We should be pretty careful with that, I think. We must follow up the runtime version closely. 
iptables rules can be missing due to a bug we caught. So, if the runtime is already a supporting one, we must immediately turn the skip out. Otherwise we may miss bugs.